src/python/bot/tasks/setup.py

```diff
@@ -93,35 +94,61 @@ def _copy_testcase_to_device_and_setup_environment(testcase,
     android.adb.run_shell_command(['chmod', '0755', device_testcase_file_path])
 
 
+def _get_application_arguments(testcase, task_name):
+  """Get application arguments to use for setting up testcase. Use minimized
+   arguments if available. For variant task, where we run a testcase against
+   another job type, use both minimized arguments and application arguments
+   from job."""
+  testcase_args = testcase.minimized_arguments
+  if not testcase_args:
+    return None
+
+  if task_name != 'variant':
+    return testcase_args
+
+  job_args = data_handler.get_value_from_job_definition(
+      testcase.job_type, 'APP_ARGS', default='')
+  job_args_list = shlex.split(job_args)
+  testcase_args_list = shlex.split(testcase_args)
+  testcase_args_filtered_list = [
+      s for s in testcase_args_list if s not in job_args_list
```

Comment:
What does `s` stand for? Maybe use `arg` or `a`

Reply:
Done!